### PR TITLE
UniversityofChicago.StandardMLofNewJersey version 110.99.2

### PR DESCRIPTION
**Pull request by simonhollingshead**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/43949)

### CODE REVIEW

File: manifests/u/UniversityofChicago/StandardMLofNewJersey/110.99.2/UniversityofChicago.StandardMLofNewJersey.installer.yaml
Comment (ItzLevvie) [inline manifests/u/UniversityofChicago/StandardMLofNewJersey/110.99.2/UniversityofChicago.StandardMLofNewJersey.installer.yaml:5]:
```suggestion
PackageVersion: 0.0.0.0
```

File: manifests/u/UniversityofChicago/StandardMLofNewJersey/110.99.2/UniversityofChicago.StandardMLofNewJersey.locale.en-US.yaml
Comment (ItzLevvie) [inline manifests/u/UniversityofChicago/StandardMLofNewJersey/110.99.2/UniversityofChicago.StandardMLofNewJersey.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 0.0.0.0
```

File: manifests/u/UniversityofChicago/StandardMLofNewJersey/110.99.2/UniversityofChicago.StandardMLofNewJersey.yaml
Comment (ItzLevvie) [inline manifests/u/UniversityofChicago/StandardMLofNewJersey/110.99.2/UniversityofChicago.StandardMLofNewJersey.yaml:5]:
```suggestion
PackageVersion: 0.0.0.0
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/u/UniversityofChicago/StandardMLofNewJersey/110.99.2/UniversityofChicago.StandardMLofNewJersey.installer.yaml
Comment (ItzLevvie) [inline manifests/u/UniversityofChicago/StandardMLofNewJersey/110.99.2/UniversityofChicago.StandardMLofNewJersey.installer.yaml:13]:
```suggestion
  ProductCode: '{F7CA0FDB-0C97-480B-A532-2A579917CFDB}'
  Scope: machine
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/u/UniversityofChicago/StandardMLofNewJersey/110.99.2/UniversityofChicago.StandardMLofNewJersey.installer.yaml
Comment (simonhollingshead) [reply manifests/u/UniversityofChicago/StandardMLofNewJersey/110.99.2/UniversityofChicago.StandardMLofNewJersey.installer.yaml:5]: I've reached out to the SMLNJ maintainers pointing out where in their build they overwrite the version string in the MSI with zeroes, and I've received a response back saying they'll look into it.  I'm going to hold off on fixing the YAML files and instead see if they can release a version where the PackageVersion actually reflects the number (otherwise updates through WinGet aren't going to work properly).

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
> I've reached out to the SMLNJ maintainers pointing out where in their build they overwrite the version string in the MSI with zeroes, and I've received a response back saying they'll look into it. I'm going to hold off on fixing the YAML files and instead see if they can release a version where the PackageVersion actually reflects the number (otherwise updates through WinGet aren't going to work properly).

You can set this PR to DRAFT until it has been fixed ;)

**Review by ItzLevvie — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by raghavthind2005 — APPROVED** [review]

**Comment by stephengillie** [comment]
Hi @simonhollingshead,

This PR looks ready to merge. Should we change it from draft to ready?

**Comment by simonhollingshead** [comment]
The maintainers have not been able to fix the version string from 0.0.0 to the real one, and this completely breaks WinGet's ability to determine when a new version is available.  I am deleting this PR.

**Pull request closed without merging**
